src/presentation/http/router/noteList.test.ts

```diff
@@ -1,116 +1,149 @@
+      },
+      url: `/notes?page=${pageNumber}`,
     });
 
-    test('Returns noteList with specified length (for last page)', async () => {
-      const portionSize = 19;
-      const pageNumber = 2;
-
-      /** create test notes for created user */
-      for (let i = 0; i < portionSize + 30; i++) {
-        await global.db.insertNote({
-          creatorId: user.id,
-        });
-      }
-
-      const response = await global.api?.fakeRequest({
-        method: 'GET',
-        headers: {
-          authorization: `Bearer ${accessToken}`,
-        },
-        url: `/notes?page=${pageNumber}`,
+    expect(response?.statusCode).toBe(200);
+
+    expect(response?.json().items).toHaveLength(portionSize);
+  });
+
+  test('Returns noteList with specified length (for last page)', async () => {
+    const portionSize = 19;
+    const pageNumber = 2;
+    let note;
+
+    /** create test notes for created user */
+    for (let i = 0; i < portionSize + 30; i++) {
+      note = await global.db.insertNote({
+        creatorId: user.id,
       });
 
-      expect(response?.statusCode).toBe(200);
+      await global.db.insertNoteVisit({
+        userId: user.id,
+        noteId: note.id,
+      });
+    }
+
+    const response = await global.api?.fakeRequest({
+      method: 'GET',
+      headers: {
+        authorization: `Bearer ${accessToken}`,
+      },
+      url: `/notes?page=${pageNumber}`,
+    });
+
+    expect(response?.statusCode).toBe(200);
 
-      expect(response?.json().items).toHaveLength(portionSize);
+    expect(response?.json().items).toHaveLength(portionSize);
+  });
+
+  test('Returns noteList with no items if it has no notes', async () => {
+    const pageNumber = 3;
+
+    const response = await global.api?.fakeRequest({
+      method: 'GET',
+      headers: {
+        authorization: `Bearer ${accessToken}`,
+      },
+      url: `/notes?page=${pageNumber}`,
     });
 
-    test('Returns noteList with no items if it has no notes', async () => {
-      const pageNumber = 3;
+    expect(response?.statusCode).toBe(200);
 
-      const response = await global.api?.fakeRequest({
-        method: 'GET',
-        headers: {
-          authorization: `Bearer ${accessToken}`,
-        },
-        url: `/notes?page=${pageNumber}`,
-      });
+    expect(response?.json()).toEqual( { items : [] } );
+    expect(response?.json().items).toHaveLength(0);
+  });
 
-      expect(response?.statusCode).toBe(200);
+  test('Returns 400 when page < 0', async () => {
+    const pageNumber = 0;
 
-      expect(response?.json()).toEqual( { items : [] } );
-      expect(response?.json().items).toHaveLength(0);
+    const response = await global.api?.fakeRequest({
+      method: 'GET',
+      headers: {
+        authorization: `Bearer ${accessToken}`,
+      },
+      url: `/notes?page=${pageNumber}`,
     });
 
-    test('Returns 400 when page < 0', async () => {
-      const pageNumber = 0;
+    expect(response?.statusCode).toBe(400);
```

Comment:
status code checking is not enough. You need to check the whole response (it should not contain notes)